packages/features/flags/config.ts

```diff
@@ -10,5 +10,6 @@ export type AppFlags = {
   workflows: boolean;
   "v2-booking-page": boolean;
   "managed-event-types": boolean;
+  newbooker: boolean;
```

Comment:
Isn't this the same as `v2-booking-page` ?

Reply:
Oh yes it is, I totally overlooked it. 
Did a quick search now though, and it doesn't seem to be used at all. There's also no migration that adds it. So there only seems to be a type floating around somewhere that isn't used. So I'm gonna delete the v2 one.